crates/bevy_math/src/bounding/bounded3d/primitive_impls.rs

```diff
@@ -535,4 +553,18 @@ mod tests {
         assert_eq!(bounding_sphere.center, translation);
         assert_eq!(bounding_sphere.radius(), 1.5);
     }
+
+    #[test]
+    fn ramp() {
+        let ramp = Ramp::new(1.0, 3.0, 4.0);
+        let translation = Vec3::new(-3.0, 1.75, 0.0);
+
+        let aabb = ramp.aabb_3d(translation, Quat::IDENTITY);
+        assert_eq!(aabb.min, Vec3::new(-3.5, 0.25, -2.0));
+        assert_eq!(aabb.max, Vec3::new(-2.5, 3.25, 2.0));
```

Comment:
Pretty sure this code doesn't work on main (since we have `Vec3A`s now), could the PR be rebased?